doc/adding-new-chains.md

```diff
@@ -0,0 +1,18 @@
+# Adding new chains
+
+This doc only describes how to add new chains on the client side (for now), so make sure it's added on `status-go` first.
```

Comment:
I'd use `chain` instead of "it's" to be clear about what we are referring to: `so make sure chain added on status-go first.`

Also [there is a doc](https://github.com/status-im/eth-rpc-proxy/blob/master/ADD_NEW_CHAINS.md) on how to add chains to rpc proxy. Optionally we can leave a link to it here, so reader has more information at hands.

Reply:
good idea. Added the link and adjusted the text.